scripts/run-browsers-smoketests.js

```diff
@@ -0,0 +1,29 @@
+const shell = require("shelljs");
+
+// set -eo pipefail
+
+console.log(`
+Test webextension-polyfill on real browsers
+===========================================
+`);
+
+if (process.platform == "win32" && process.env.EDGE_DRIVER_PATH) {
+  process.env.PATH += `;${process.env.EDGE_DRIVER_PATH}`;
```

Comment:
Where is this environment variable documented? If it is only used as a convenience to set the location of the Edge driver, shouldn't this be in the test-integration implementation?

If the line is to be kept, let's move this line into the block with the same condition at the end of this file.